public_app/components/BigImg.js

```diff
@@ -0,0 +1,62 @@
+import React        from 'react';
+import Carousel     from 'nuka-carousel';
+import MovieItem    from './MovieItem';
+import axios        from 'axios';
+import { Link }     from "react-router";
+
+
+
+class BigImg extends React.Component {
+    constructor (props) {
+        super(props);
+        this.state = {
+            items: [],
+            id: '',
+            searchInput: '',
+            searchTitle: [],
+            searchYear: []
+        };
+
+
+        axios.get("api/movies").then((response) => {
+            this.setState({
+                items: response.data
+            });
+        });
+
+    }
+
+
+    render (){
+        let searchTrue = this.props.inputFieldValue;
+        console.log(searchTrue);
+        return (
+            <div className="big-img-container">
```

Comment:
Should try to use css coding conventions like BEM http://getbem.com/introduction/